Automattic/jetpack
# CLI Generate: Create mirror repo

WIP: Currently, adds and tests function to check for existing repo. When running via GH Actions, it uses the GH action to check for private repos. If running `yarn test` locally, it'll ask and store your GitHub Personal Access Token.

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* `cd tools/cli && yarn install`
* `yarn test`

It'll ask you for a Github Personal Token on first run. See https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token

#### Proposed changelog entry for your changes:


*
